### PR TITLE
refactoring the tag center from HTML4 to HTML5

### PR DESCRIPTION
**Pull request by MohamedZaghdoudi**
This PR resolves issue #3235 "Use of "center" tag" : 

by switching the tag "center" from HTML4 to HTML5 using the CSS alternative <div style="text-align:center">.

---
Internal Tracking ID: [EXPOSUREAPP-14424](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14424)

### CODE REVIEW

File: blog/2020-12-16-cwa-release-1-9/index.md
Comment (MikeMcC399) [inline blog/2020-12-16-cwa-release-1-9/index.md:21]: The end tag `</center>` has been converted to the start tag `<div>`. This is wrong and causes Google chrome to then center the following text.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Please run this locally (`npm start`) and look at the output e.g. http://localhost:8000/en/blog/2020-12-16-corona-warn-app-version-1-9/

image

Only the images should be centered, not the following text.

In multiple places where there should be a `</div>` end tag there is a `<div>` start tag instead.

**Comment by MohamedZaghdoudi** [comment]
@MikeMcC399   I will delete this pull request and upload again

File: blog/2020-12-16-cwa-release-1-9/index.md
Comment (brianebeling) [reply blog/2020-12-16-cwa-release-1-9/index.md]: Seems to be resolved

**Review by brianebeling — APPROVED** [review]
Tested locally, LGTM

File: blog/2020-12-16-cwa-release-1-9/index.md
Comment (MikeMcC399) [inline blog/2020-12-16-cwa-release-1-9/index.md:21]: `style="align: center"` causes
"Error: CSS: align: Property align doesn't exist."

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
This looks a lot better now!

Can you take a look at `style="align: center"` and see if it can be removed everywhere? It is not defined and it seems redundant.

**Review by MikeMcC399 — APPROVED** [review]
@MohamedZaghdoudi 

Thank you for the big clean up! It looks correct now.

Will you squash your commits before this PR is merged?

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi 

Did you consider using `class="text-center"` rather than `style="text-align:center"` here?

**Comment by brianebeling** [comment]
I think we can merge this after the final adjustment has been implemented, in order to move a bit faster. 😄

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi 

You seem to be having conflicts with previous PRs and with rebasing. You probably need to revert (or undo) your last commit and rebase on the current master branch.

**Comment by brianebeling** [comment]
@MohamedZaghdoudi Let me know if you need help!

**Comment by brianebeling** [comment]
Sorry for the slight mess. Everything should be up to date now. I decided the fastest way to learn git is to break things. Refer to the file changes for actual changes, please.

File: src/data/event-qr-code-guide.json
Comment (brianebeling) [inline src/data/event-qr-code-guide.json:14]: Center has been swapped with <div> but without styles, this will not be centered

Comment (MikeMcC399) [reply src/data/event-qr-code-guide.json:14]: @brianebeling 

Moving some of the text to left-alignment in this case is an improvement. The center formatting was quite strange previously.

**Comment by brianebeling** [comment]
Let me know once this is reviewable again 😄

**Comment by MikeMcC399** [comment]
@brianebeling 

I suggest **not** to continue to center the text in https://www.coronawarn.app/en/event-qr-code-guide/. It looks very odd at the moment, independently of this PR.

If you really want to use text-center in json source then you can't use double quotes on their own. Either
- `class='text-center'` with single quotes or
- `class=\"text-center\"` with escaped double quotes

In general it is better to at least use `npm run build` before pushing to the repository, better still, run `npm start` to view the outcome and finally `npm test` to make sure that no unexpected errors have been introduced.

**Comment by brianebeling** [comment]
> Brian-J. Ebeling
> 
> I suggest **not** to continue to center the text in [coronawarn.app/en/event-qr-code-guide](https://www.coronawarn.app/en/event-qr-code-guide/). It looks very odd at the moment, independently of this PR.
> 
> If you really want to use text-center in json source then you can't use double quotes on their own. Either
> 
> * `class='text-center'` with single quotes or
> * `class=\"text-center\"` with escaped double quotes
> 
> In general it is better to at least use `npm run build` before pushing to the repository, better still, run `npm start` to view the outcome and finally `npm test` to make sure that no unexpected errors have been introduced.

Sure, not centering is a great option. If its an improvement lets keep it. It was not my intention to suggest that we should a hundred percent keep it centered. Verbally I instructed @MohamedZaghdoudi to compare both approaches and let us know which looks better. It'd be best to address @MohamedZaghdoudi going forward with this ticket.

**Comment by MikeMcC399** [comment]
@MohamedZaghdoudi  / @brianebeling 

This PR affects a lot of files and is taking quite some time. I am wondering if we should reduce the scope and split it up to simplify reviewing and approving it.

For instance:
- Only target the latest blog post 2.28 https://www.coronawarn.app/en/blog/2022-10-26-cwa-2-28/ and the parts of the blogs which appear in the overview on https://www.coronawarn.app/en/blog/, making no other changes to the historical blogs
- Split the changes to the science blog https://www.coronawarn.app/en/science/2022-03-03-science-blog-5/ into a separate PR
- Split the changes for https://www.coronawarn.app/en/event-qr-code-guide/ into a separate PR

(same for German links as well)

**Comment by brianebeling** [comment]
@MikeMcC399 Lets wait for @MohamedZaghdoudi response as this PR is his responsibility. Currently he is sick, but he will get back to the task once he is back in action. I don't mind the big list of file changes as long as the purpose is clear. In the future it might be a good idea to try smaller PRs though. Anyways, lets wait for Mohameds answer. Thanks for bringing it up!

**Comment by MikeMcC399** [comment]
@brianebeling I'm sorry to hear that @MohamedZaghdoudi is sick! As you say, we should wait in that case.

**Review by larswmh — CHANGES_REQUESTED** [review]
To me, this PR looks good now. 

The only two files that would still need to be adressed are [blog\2022-10-26-cwa-2-28\index_de.md](https://github.com/corona-warn-app/cwa-website/blob/master/blog/2022-10-26-cwa-2-28/index_de.md) & [blog\2022-10-26-cwa-2-28\index.md](https://github.com/corona-warn-app/cwa-website/blob/master/blog/2022-10-26-cwa-2-28/index.md)

**Comment by MikeMcC399** [comment]
@larswmh 

The centering of the text, especially output from a terminal window, on https://www.coronawarn.app/en/event-qr-code-guide/ looks very strange. It should be left-justified. This could however be dealt with in a separate issue / PR. See my previous https://github.com/corona-warn-app/cwa-website/pull/3241#issuecomment-1334048932 on the subject.

**Comment by larswmh** [comment]
@MikeMcC399 

> The centering of the text, especially output from a terminal window, on https://www.coronawarn.app/en/event-qr-code-guide/ looks very strange. It should be left-justified. This could however be dealt with in a separate issue / PR. See my previous [#3241 (comment)](https://github.com/corona-warn-app/cwa-website/pull/3241#issuecomment-1334048932) on the subject.

I agree that it does look strange. Since this was a change that was not introduced in this PR though, I would deal with this in a seperate issue / PR as you suggested.

File: blog/2020-12-16-cwa-release-1-9/index.md
Comment (larswmh) [reply blog/2020-12-16-cwa-release-1-9/index.md]: As of the most recent version of this PR, this is fixed. Thanks for the information

**Review by larswmh — APPROVED** [review]
Now that the latest blog has been updated accordingly, I think this is ready to be merged

**Pull request merged**
